{"name": "Логин с неверным логином или паролем", "status": "passed", "description": "Этот тест проверяет, что при неверных данных логина возвращается ошибка.", "steps": [{"name": "Отправка запроса с неправильными данными", "status": "passed", "start": 1744925092544, "stop": 1744925093030}, {"name": "Проверка, что возвращается статус-код 401", "status": "passed", "start": 1744925093030, "stop": 1744925093030}, {"name": "Проверка, что сообщение об ошибке корректно", "status": "passed", "start": 1744925093030, "stop": 1744925093031}], "parameters": [{"name": "email", "value": "'wrong@example.com'"}, {"name": "password", "value": "'correctpass'"}], "start": 1744925092544, "stop": 1744925093031, "uuid": "f05ce545-6a5f-4698-b951-b32188997cc5", "historyId": "3d011ab97254a167321df6cbc515a60e", "testCaseId": "8f9cda47c51ec7bbc26bb76800a7f0aa", "fullName": "tests.test_user_login#test_login_with_invalid_credentials", "labels": [{"name": "feature", "value": "Пользователь"}, {"name": "story", "value": "Логин"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "host", "value": "MacBook-Pro-George.local"}, {"name": "thread", "value": "39352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}